### DISCUSSION TITLE
Mapping DNS record for Web3Modal v2.0

### DISCUSSION THREAD

**Discussion by atlos-io** [WalletConnect/walletconnect-monorepo#2974] [Developer Support] [answered]
Hello,

We distribute our app as a widget that can be added to any website. It's used for accepting crypto payments. We are experiencing this error:

`Failed to execute 'postMessage' on 'DOMWindow'`

I understand that in order to fix it, we need to ensure we are the rightful domain owner of all our clients' websites, which we are not (https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/dapps/dapp-checklist).

Does it mean we cannot use Web3Modal in our app any more?

Thank you

**Comment by ganchoradkov** [comment, marked as answer]
Hey @atlos-io, please disregard this error. It means a POST message was made to an iframe that's still in a loading state. Browsers have built-in retry functionality so the POST would succeed eventually. Also, It shouldn't cause any disruption in the usage of any of the SDKs
